resource/plugins/unreal/python/publisher/collectors/unreal_asset_publisher_collector.py

```diff
@@ -18,10 +18,22 @@ class UnrealAssetPublisherCollectorPlugin(
 
     plugin_name = 'unreal_asset_publisher_collector'
 
+    def fetch(self, context_data=None, data=None, options=None):
+        '''Fetch the selected asset from content browser'''
+
+        selected_asset_data = (
+            unreal.EditorUtilityLibrary.get_selected_asset_data()
+        )
+
+        if len(selected_asset_data) == 0:
+            return ''
+
+        return str(selected_asset_data[0].package_name)
+
     def run(self, context_data=None, data=None, options=None):
         '''Pass on collected Unreal assets to publish'''
-        unreal_objects = options.get('collected_objects', [])
-        return unreal_objects
+        asset_path = options.get('asset')
```

Comment:
I would leave collected_objects as all the other collectors to maintain consistency.